CMSIS-OS/ChibiOS/ST_NUCLEO64_F411RE_NF/nanoCLR/STM32F411xE_CLR-DEBUG.ld

```diff
@@ -12,7 +12,7 @@
  */
 MEMORY
 {
-    flash       : org = 0x08004000, len = 512k - 16k - 256k   /* flash size less the space reserved for nanoBooter and application deployment*/
+    flash       : org = 0x08004000, len = 512k - 32k - 256k   /* flash size less the space reserved for nanoBooter and application deployment*/
```

Comment:
Shouldn't this be 0x08008000?

Reply:
YES but forgot to commit late yesterday ,
so i commit today morning